src/Assets/Images/image.js

```diff
@@ -19,6 +19,13 @@ import women1 from "./Women/women1.webp";
 import women2 from "./Women/women2.webp";
 import women3 from "./Women/women3.webp";
 import women4 from "./Women/women4.webp";
+import banner1 from "./Banner/purple-banner.jpg";
```

Comment:
Suggestion: You can import images as an object to avoid multiple imports. [Like](https://github.com/zoef-yt/react-ecom/blob/6778f9d8da601fe031c48c60078c82a25a399e7d/src/assets/images/index.js#L1)